assets/src/edit-story/components/canvas/multiSelectionMovable.js

```diff
@@ -29,6 +29,7 @@ import objectWithout from '../../utils/objectWithout';
 import { useTransform } from '../transform';
 import { useUnits } from '../../units';
 import { getDefinitionForType } from '../../elements';
+import { useGlobalKeyDownEffect, useGlobalKeyUpEffect } from '../keyboard';
```

Comment:
Since single- and multiselection seem to share the snapping props, would it perhaps make sense to create the key handler in `Selection` and set the snapping props there instead to pass down? Or alternatively use an util for getting the snapping props. Thoughts?

Reply:
I think there's a lot more we can refactor between the two, I'd leave this for another PR